data/fleets.txt

```diff
@@ -17,50 +17,27 @@ fleet "Small Southern Merchants"
 		timid frugal appeasing
 	variant 60
 		"Shuttle"
```

Comment:
Another question is, should we create a `variant` for every ship, even if that ship doesn't have any variants? For example, a "Shuttle" variant consisting of only "Shuttle". This would mean that all vanilla fleet variants would consist entirely of nested variants. The reason for doing this would be so that any future ship variant additions would require only to add to the `variant` for that ship in order to include it in any fleets, instead of perhaps having to figure out if said ship is already in a variant first, then creating the variant and placing it wherever needed if necessary. It's basically a question of doing the work now, even if nothing comes of it in the future (e.g. there may never be a Shuttle ship variant aside from the base model), or doing the work later.

Reply:
No, I don't think it makes sense to do this now. We might also just want to store top-level fleet variants and passed events in savegames so that such updates can also be done in the future for existing savegames.
If we go that direction, then defining such variants at this moment would be a lot of work with no return-on-investment.